src/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { ActorInput } from './types.js';
+
+/**
+ * Validates Input provided by user
+ * @throws Will throw an error if input is null,
+ * or it does not contain html/templateData or if templateData does not have any keys. (It does not make sense to generate OG image without dynamic values)
+ */
+export const validateInput = (actorInput: ActorInput | null) => {
```

Comment:
```suggestion
export const validateInput = (actorInput: ActorInput | null): asserts actorInput is ActorInput => {
```

Reply:
This way you won't have to do the `actorInput!` assertion later on, because if you use this function, you'll know that the argument is not null